corehq/apps/hqmedia/views.py

```diff
@@ -270,10 +272,101 @@ def update_multimedia_paths(request, domain, app_id):
             'error': _("Could not find file. Please re-upload file.")
         })
 
-    # TODO: update paths
```

Comment:
Can you break this down into small logical units/methods? Would help better understand and naming would give some documentation by default.

Reply:
yeah, I'll take a look at this

Reply:
Did a bunch of reorganization, starting with 5ec9ead6aff652d4ab3d856745f1d1299d6f6f64 The overall change is to move all of the renaming logic into hqmedia.models, and to break HQMediaMixin into smaller classes: one for apps, one for modules, and one for forms.